### PR TITLE
Draft / WIP: Just figure out how to ad a media type to jupyter repl output formats

### PR DESCRIPTION
**Pull request by NyanHelsing**
Closes #ISSUE

Release Notes:

- Figuring out how to add a media type to jupyter repl outputs.
- Plan to use Wry or maybe Servo to render a webview.

### Or...

Closes #ISSUE

Release Notes:

- N/A

Deps (Ubuntu):

```
sudo apt install libjavascriptcoregtk-4.1-dev
sudo apt install libpango1.0-dev
sudo apt install libgdk-pixbuf2.0-dev
sudo apt install libgtk-3-dev
sudo apt install libsoup-3.0-dev
sudo apt install libwebkit2gtk-4.1-dev
```

### CODE REVIEW

File: crates/ui/src/components/web_view.rs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+struct WebViewTest {
+    views: Vec<Arc<WebView>>,
+}
+
+impl WebViewTest {
+    fn new(num_views: usize, handle: &dyn HasWindowHandle) -> Self {
```
Comment (NyanHelsing) [inline crates/ui/src/components/web_view.rs:6]: maybe i will make this an element? i also may just leave it only as a view in the outputs.rs file.

**Comment by NyanHelsing** [comment]
> We require contributors to sign our Contributor License Agreement, and we don't have @me-nyanhelsing on file. You can sign our CLA at https://zed.dev/cla. Once you've signed, post a comment here that says '@cla-bot check'.

@cla-bot check

**Comment by NyanHelsing** [comment]
> This PR does not look well, as even the CLA issues were not solved, hence will close it.

Hey a couple things:

- I actually did sign this; I think maybe a bug in the bot?
Screenshot from 2024-09-11 09-12-37

note how the bot is asking for @me-nyanhelsing  but my username is actually @NyanHelsing .

- I marked this as a draft because this isn't ready to merge; I typically use PRs as a way to keep notes for myself on diffs, and to share progress on my work.

**Comment by SomeoneToIgnore** [comment]
https://github.com/zed-industries/zed/pull/17004/commits clearly shows GitHub metadata related to @me-nyanhelsing 
You must have used a different e-mail for signing commits and in your GH profile — that discrepancy confuses both the bot and GitHub.

**Pull request closed without merging**